infrastructure/main.tf

```diff
@@ -7,6 +7,10 @@ locals {
   s2s_rg              = "rpe-service-auth-provider-${local.local_env}"
   s2s_url             = "http://${local.s2s_rg}.service.core-compute-${local.local_env}.internal"
   ccd_api_url         = "http://ccd-data-store-api-${local.local_env}.service.core-compute-${local.local_env}.internal"
+
+  previewVaultName    = "${var.product}-aat"
```

Comment:
preview is not needed ?

Reply:
removed it now. 
I'll test the pr on sandbox